lib/lib.go

```diff
@@ -1,8 +1,22 @@
 package lib
 
+import (
+	"fmt"
+	"runtime"
+)
+
 // Logger is the interface to support swappable loggers.
 type Logger interface {
 	Info(string)
+	Error(string)
+}
```

Comment:
We can't add Error without a breaking change, let's discuss this in the v2.0 roadmap issue.